lib/koans/10_pattern_matching.ex

```diff
@@ -7,24 +7,6 @@ defmodule PatternMatching do
     assert match?(1, ___)
   end
 
-  koan "A value can be bound to a variable" do
-    a = 1
-    assert a == ___
-  end
-
-  koan "A variable can be rebound" do
-    a = 1
-    a = 2
-    assert a == ___
-  end
-
-  koan "A variable can be pinned to prevent it from being rebounded" do
-    a = 1
-    assert_raise MatchError, fn() ->
```

Comment:
I do still sort of like this to illustrate what happens when a pinned match fails. Maybe introduce this before some of the more advanced examples?